internal/utils/templates/init_config.toml

```diff
@@ -23,6 +23,14 @@ shadow_port = 54320
 # server_version;` on the remote database to check.
 major_version = 15
 
+[db.pooler]
+enabled = false
+port = 54329
+pool_mode = "transaction"
+default_pool_size = 20
+ignore_startup_parameters = ["extra_float_digits"]
+max_client_conn = 100
```

Comment:
Does supavisor support these? We'll probably move away from pgbouncer at some point

Reply:
I doubt users can configure these on hosted platform after supavisor GA since it's a multi-tenant service. These configs are probably still relevant for testing in local development though.

Reply:
How come this is relevant for local development when it isn't on hosted?

Reply:
Supavisor does support [per-tenant config](https://github.com/supabase/supavisor/wiki/Installation-and-Usage#addupdate-tenant), but we can omit `ignore_startup_parameters` for now

Reply:
Sounds good!